src/main/java/org/terasology/moduletestingenvironment/fixtures/ItemDropSpy.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.moduletestingenvironment.fixtures;
+
+import org.terasology.context.Context;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.event.internal.EventSystem;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.logic.inventory.events.DropItemEvent;
+import org.terasology.moduletestingenvironment.ModuleTestingEnvironment;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Test fixture for recording item drops.
+ * <p>
+ * Two usage patterns are supported.  A test may instantiate an {@code ItemDropSpy}, execute some
+ * code, and then examine the list of drops provided by {@link #getDrops()}.  In order to handle
+ * registration and un-registration of the event handler listening for item drops, this should be
+ * done in a try-with-resources block ({@code ItemDropSpy} is {@link AutoCloseable}):
+ * {@code
+ * try (ItemDropSpy spy = new ItemDropSpy(getHostContext())) {
+ *   dropExampleItems();
+ *   assertTrue(spy.getDrops().exampleItemsAreCorrect());
+ * }
+ * }
+ * <p>
+ * Alternatively, a test may call the static method {@link #collectDrops(Context, Runnable)}, which
+ * may be easier to understand, especially if the code being run is a single method.  In this style,
+ * the previous example would look like this:
+ * {@code
+ *   List<EntityRef> drops = ItemDropSpy.collectDrops(getHostContext(), ::dropExampleItems);
+ *   assertTrue(drops.exampleItemsAreCorrect());
+ * }
+ */
+public class ItemDropSpy extends BaseComponentSystem implements AutoCloseable {
+
+  private EventSystem eventSystem;
```

Comment:
Am I seeing it right than indents for this PR is 2 spaces per indent instead of 4? We're still 4 as far as the convention goes, although I've been wondering about 2 (Google Style). Probably not a thing to change with this PR tho :-)

Reply:
Simple enough to fix.  Done.